conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureDoubleList.java

```diff
@@ -69,4 +80,22 @@ public void clear() {
     public Double set(int index, Double element) {
         return delegate.set(index, element);
     }
+
+    static final class Serializer extends JsonSerializer<ConjureDoubleList> {
+        @Override
+        public void serialize(ConjureDoubleList val, JsonGenerator gen, SerializerProvider _serializer)
+                throws IOException {
+            double[] doubles = val.delegate.toArray();
+            gen.writeArray(doubles, 0, doubles.length);
+        }
+    }
+
+    static final class Deserializer extends JsonDeserializer<ConjureDoubleList> {
+        @Override
+        public ConjureDoubleList deserialize(JsonParser parser, DeserializationContext _ctxt) throws IOException {
+            // Avoid making a copy of the value from jackson
+            double[] doubles = parser.readValueAs(double[].class);
+            return new ConjureDoubleList(new DoubleArrayList(doubles));
+        }
+    }
```

Comment:
I don't think these are used anywhere in practice -- Jackson isn't  provided the `ConjureDoubleList` type as a deserialization target, only `List<Double>`. On the serialization side, the constructed objects use `Collections.unmodifiableList` to wrap these specialized types, losing this specialization.

Reply:
I removed deserialization for now, will reintroduce later.